pkg/cidata/template_test.go

```diff
@@ -30,11 +30,11 @@ func TestConfig(t *testing.T) {
 
 func TestConfigCACerts(t *testing.T) {
 	args := &TemplateArgs{
-		Name:  "default",
-		User:  "foo",
-		UID:   501,
-		GECOS: "Foo",
-		Home:  "/home/foo.linux",
+		Name:    "default",
+		User:    "foo",
+		UID:     501,
```

Comment:
Does the yaml library support unmarshalling this number literal to *string?

LGTM then, feel free to merge it

Reply:
Somewhat ironically the `UID` field in the cidata template is an `int`, and then gets turned into a string again in `user-data` with `uid: "{{.UID}}"`. 😄 

Anyways, I've change `uid` to an `int` in `lima.yaml` because I don't like changing the schema again later.

So please approve again once everything is green again.

Reply:
Ok, changed it one more time to `uint32` to avoid CodeQL errors...

Reply:
> LGTM then, feel free to merge it

You have requested changes; please formally approve. I don't want to merge without the explicit approval because it would hurt our OSSF score...